README.md

```diff
@@ -148,6 +148,8 @@ apt::source { 'puppetlabs':
 },
 ~~~
 
+To use a repository served over HTTPS, `include ::apt::transport_https`.
```

Comment:
Could you document this class in the same manner as the other classes in the README? THanks!